src/main/webapp/app/overview/course-dashboard/course-dashboard.component.ts

```diff
@@ -0,0 +1,190 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { CourseStorageService } from 'app/course/manage/course-storage.service';
+import { Subscription, forkJoin } from 'rxjs';
+import { Exercise, IncludedInOverallScore } from 'app/entities/exercise.model';
+import { ActivatedRoute } from '@angular/router';
+import { TranslateService } from '@ngx-translate/core';
+import { Course } from 'app/entities/course.model';
+import { CourseScores } from 'app/course/course-scores/course-scores';
+import { ScoreType } from 'app/shared/constants/score-type.constants';
+import { Competency } from 'app/entities/competency.model';
+import { onError } from 'app/shared/util/global.utils';
+import { CompetencyService } from 'app/course/competencies/competency.service';
+import { HttpErrorResponse } from '@angular/common/http';
+import { AlertService } from 'app/core/util/alert.service';
+import dayjs from 'dayjs/esm';
+import { ScoresStorageService } from 'app/course/course-scores/scores-storage.service';
+import { FeatureToggle } from 'app/shared/feature-toggle/feature-toggle.service';
+
+@Component({
+    selector: 'jhi-course-dashboard',
+    templateUrl: './course-dashboard.component.html',
+    styleUrl: './course-dashboard.component.scss',
```

Comment:
Since the associated file is empty, I think this line and that file can be deleted.

Reply:
Deleted